src/Analyzers/CSharp/Tests/SimplifyLinqExpressions/SimplifyLinqExpressionsTests.cs

```diff
@@ -0,0 +1,681 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.SimplifyLinqExpressions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Analyzers.UnitTests.SimplifyLinqExpressions
+{
+    public partial class SimplifyLinqExpressionsTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
```

Comment:
use new unit testing harness here.

Reply:
@CyrusNajmabadi can you point them at an existing test class that uses the new test harness?

---
In reply to: [468118805](https://github.com/dotnet/roslyn/pull/46672#discussion_r468118805) [](ancestors = 468118805)